stripe-ui-core/src/test/java/com/stripe/android/uicore/elements/PhoneNumberFormatterTest.kt

```diff
@@ -15,9 +15,10 @@ internal class PhoneNumberFormatterTest {
         assertThat(formatter.format("123456")).isEqualTo("(123) 456")
         assertThat(formatter.format("1234567")).isEqualTo("(123) 456-7")
         assertThat(formatter.format("1234567890")).isEqualTo("(123) 456-7890")
+        // Re-enable this test when maximum length are properly defined for supported countries
         // prefix has 1 digit so full number must be at most 14 digits
-        assertThat(formatter.format("12345asdfg678901234567890"))
-            .isEqualTo("(123) 456-7890")
+        // assertThat(formatter.format("12345asdfg678901234567890"))
```

Comment:
I vote we just remove this, and the associated comment. We can look at git history if we decide to implement this in the future. But AFAIK, we don't have plans to do so.